cmd/worker/main.go

```diff
@@ -227,10 +229,24 @@ func messageLoop(ctx context.Context, subURL, packagesBucket, notificationTopicU
 			// All subsequent receive calls will return the same error, so we bail out.
 			return fmt.Errorf("error receiving message: %w", err)
 		}
+		me := extender.Start(ctx, msg, func() {
+			log.Info("Message Ack deadline extended", "message_id", msg.LoggableID, "message_meta", msg.Metadata)
+		})
 
 		if err := handleMessage(ctx, msg, pkgsBkt, resultsBuckets, imageSpec, notificationTopic); err != nil {
 			log.Error("Failed to process message",
 				"error", err)
```

Comment:
nit: remove the linebreak in this function call to be consistent with the other log calls

Reply:
This is a callback method in the call to `Start()` I'm not sure Go's formatter likes removing those linebreaks.

Reply:
Oh iThe commit introducing the change is [this one]( https://github.com/composer/getcomposer.org/commit/70527179915d55b3811bebaec55926afd331091b)

Reply:
Oh, I didn't see the callback part, no worries then